src/angular/13-nested-routes/nested-route.html

```diff
@@ -60,13 +60,13 @@
   template: `
   <ul class="nav">
     <li routerLinkActive="active">
-      <a routerLink="/about" >About</a>
+      <a routerLink="/about">About</a>
     </li>
     <li routerLinkActive="active">
       <a [routerLink]="['/users', userId]">User 1</a>
     </li>
   </ul>
-  <router-outlet></router-outlet>
+  <router-outlet />
```

Comment:
I believe this might break the example as it is loaded in codepen and references older versions that didn't support self closing tags on angular components

Reply:
Thank you! I am reverting all of these. I’ll add something in our backlog to update these code samples to the latest version of Angular.